portfolio-product/name.abuchen.portfolio.distro.product

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <?pde version="3.5"?>
 
-<product name="Portfolio Performance" uid="name.abuchen.portfolio.distro.product" id="name.abuchen.portfolio.bootstrap.product" application="org.eclipse.e4.ui.workbench.swt.E4Application" version="0.69.1.qualifier" useFeatures="true" includeLaunchers="true">
+<product name="Portfolio Performance" uid="name.abuchen.portfolio.distro.product" id="name.abuchen.portfolio.bootstrap.product" application="org.eclipse.e4.ui.workbench.swt.E4Application" version="0.69.1.qualifier" type="features" includeLaunchers="true" autoIncludeRequirements="true">
```

Comment:
Looks like the product file format has changed.
I checked: the ```autoIncludeRequirements``` is the default Tycho behavior so I do not expect changes in the Maven build.

But it makes sense to update it.